packages/editor/src/pages/EditorPage.tsx

```diff
@@ -23,26 +23,18 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
 Infinite Reality Engine. All Rights Reserved.
 */
 
-import { PopoverState } from '@ir-engine/client-core/src/common/services/PopoverState'
 import '@ir-engine/engine/src/EngineModule'
 import { getMutableState, useHookstate, useImmediateEffect } from '@ir-engine/hyperflux'
 import { loadEngineInjection } from '@ir-engine/projects/loadEngineInjection'
 import { EngineState } from '@ir-engine/spatial/src/EngineState'
-import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
-import Modal from '@ir-engine/ui/src/primitives/tailwind/Modal'
+import { useUnsupported } from '@theinfinitereality/irpro-multitenancy/components/hooks/useUnsupported'
```

Comment:
You cannot import from a project into the core. You need to move the implementation to the core

Reply:
^

Reply:
done 163a8936f67b5aa7e48e947962d4b56d9ea3f56d